Source/towners.cpp

```diff
@@ -630,6 +630,9 @@ void TalkToFarmer(PlayerStruct &player, TownerStruct &farmer)
 		if (gbIsMultiplayer)
 			NetSendCmdQuest(true, Q_FARMER);
 		break;
+	case QUEST_HIVE_DONE:
+		// do nothing
+		return;

```

Comment:
```suggestion
		break;
```

Reply:
The only place in this whole file where `return` is used in a switch is the `TalkToGirl()` function. Personally, I would very much prefer `break` than `return` here.

Reply:
Oh - I mislooked here - I interpreted as adding a `break` after `return` (which is enforeced by some code styles) - Sorry for confusion. I see - reakng here would do the same

Reply:
Sounds like a terribly redundant code style since it would be unreachable code :D